zerver/lib/actions.py

```diff
@@ -2010,6 +2011,20 @@ def do_change_left_side_userlist(user_profile, setting_value, log=True):
         log_event(event)
     send_event(event, [user_profile.id])
 
+def mark_user_star_backend(user_profile, email):
```

Comment:
For consistency, name actions like this `do_update_user_flag` to mirror `do_update_message_flags`.